src/chapters/introduction-and-setup/setup.rst

```diff
@@ -18,10 +18,9 @@ this course, so it's time to get familiar with some of the basics.
 
 .. admonition:: Note
 
-   Visual Studio IDE is not the same application as the text editor called Visual Studio Code. 
+   Visual Studio is not the same application as the text editor called Visual Studio Code. 
```

Comment:
I see that you removed "IDE" from this page.  Could you share your thinking behind this move?  

For context, in chapter 1, we wanted to make a clear distinction between Visual Studio Code and Visual Studio.  We felt that tacking on "IDE" helped emphasize this.

Reply:
My thought was purely that no one actually refers to Visual Studio as Visual Studio IDE.  It's perfectly fine, it just struck me as odd.

Reply:
Got it.  Let's keep "IDE" in place for chapter 1.  If it pops up in later chapters, then we can remove it.